setup.py

```diff
@@ -26,6 +26,7 @@
     install_requires=open('requirements.txt').read().splitlines(),
     entry_points=dict(console_scripts=[
         'odoo-module-migrate=odoo_module_migrate.__main__:main',
+        'odoo-module-squasher=odoo_module_migrate.squasher:main',
```

Comment:
how ! you can use this new feature as a standalone, without migration process. nice !

Reply:
In fact it's not exactly usable in stadalone, but I can do it.

The current "odoo-module-squasher" is used as an arg to replace the TEXT editor for the git rebase

If you run 

GIT_EDITOR=odoo-module-squasher GIT_SEQUENCE_EDITOR=odoo-module-squasher git rebase  -i 14.0


Instead of have to edit with vim (or whatever text) my python script will do it.

It's the simpliest solution that I have found.